### DISCUSSION TITLE
mpvnext autostart

### DISCUSSION THREAD

**Discussion by Bush-cat** [jellyfin/jellyfin-mpv-shim#392] [Q&A]
Hello, I'm using mpvnet for some integrations but the autostart is differrent from normal mpv, mpv only starts when jellyfin starts playback while mpvnet starts when you start the mpv shim, if you close mpv after the shim started it, jellyfin does not stop playback and it can't be stopped anymore and you have to restart the shim.

Is it maybe possible with play_cmd, stop_cnd and such to get the original functionality back?

**Comment by dola321** [comment]
i have the same question
